pkgs/by-name/be/better-control/package.nix

```
@@ -0,0 +1,102 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  gtk3,
+  networkmanager,
+  bluez,
+  pipewire,
+  brightnessctl,
+  power-profiles-daemon,
+  gammastep,
+  libpulseaudio,
+  pulseaudio,
+  desktop-file-utils,
+  wrapGAppsHook4,
+  gobject-introspection,
+  usbguard,
+  upower,
+  nix-update-script,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "better-control";
+  version = "v6.11.4";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "quantumvoid0";
+    repo = "better-control";
+    tag = version;
+    hash = "sha256-i+r0iPeSG2a8P2oDvm7QS+Y1ok117K6gnC7EVJuKgIs=";
+  };
+
+  nativeBuildInputs = [
+    desktop-file-utils
+    wrapGAppsHook4
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    gtk3
+    libpulseaudio
+  ];
+
+  dependencies =
+    [
+      networkmanager
+      bluez
+      pipewire
+      brightnessctl
+      power-profiles-daemon
+      gammastep
+      pulseaudio
+      usbguard
+      upower
+    ]
+    ++ (with python3Packages; [
+      pygobject3
+      dbus-python
+      pydbus
+      psutil
+      qrcode
+      requests
+      setproctitle
+      pillow
+      pycairo
+    ]);
+
+  makeFlags = [ "PREFIX=${placeholder "out"}" ];
+
+  dontWrapGApps = true;
+
+  makeWrapperArgs = [ "\${gappsWrapperArgs[@]}" ];
+
+  postInstall = ''
+    rm $out/bin/betterctl
+    chmod +x $out/share/better-control/better_control.py
+    patchShebangs --build $out/bin
```

```suggestion
    patchShebangs --host $out/bin
```

Should be host? Since these shebang lines would be used at runtime not build time.

---

Yeah this is just for launching the script and I think this should be fine instead of using patchShebangs.
```
    substituteInPlace $out/bin/* \
      --replace-fail "/bin/bash" "/usr/bin/env bash" \
      --replace-fail "python3 " ""
```